addons/project/wizard/project_share_wizard.py

```diff
@@ -12,6 +12,9 @@ class ProjectShareWizard(models.TransientModel):
     @api.model
     def default_get(self, fields):
         result = super().default_get(fields)
+        if 'default_project_id' in self._context:
+            result['res_id'] = self._context['default_project_id']
+            result['share_link'] = None
```

Comment:
I don't remember if it was a bug you found or if it was for the action that you have to display in the help message in the Project collaborators action.

Reply:
It's not a bug, it's a leftover from the attempt when trying to add the button to the no content helper which was at the end of the day dropped. I'll remove this now